src/com/google/javascript/jscomp/parsing/IRFactory.java

```diff
@@ -470,6 +474,22 @@ private void validateJsDoc(Node n) {
     validateTypeAnnotations(n);
     validateFunctionJsDoc(n);
     validateMsgJsDoc(n);
+    validateDeprecatedJsDoc(n);
+  }
+
+  /**
+   * Checks that deprecated annotations such as @expose are not present
+   */
+  private void validateDeprecatedJsDoc(Node n) {
```

Comment:
add a unit test for this in
https://github.com/google/closure-compiler/blob/master/test/com/google/javascript/jscomp/parsing/NewParserTest.java